contracts/user-contracts/src/main/java/org/java/mentorship/contracts/user/dto/User.java

```diff
@@ -0,0 +1,9 @@
+package org.java.mentorship.contracts.user.dto;
+
+import lombok.Data;
+
+@Data
+public class User {
```

Comment:
Contract Data Transfer Object

The services should return those objects on their endpoints